websites/Q/Qobuz/presence.ts

```diff
@@ -0,0 +1,120 @@
+const presence = new Presence({
+  clientId: "921861694190407730"
+});
+
+async function getStrings() {
+  return presence.getStrings(
+    {
+      play: "general.playing",
+      pause: "general.paused",
+      viewAlbum: "general.buttonViewAlbum",
+      viewPlaylist: "general.buttonViewPlaylist"
+    },
+    await presence.getSetting("lang").catch(() => "en")
+  );
+}
+
+(async () => {
+  let strings = await getStrings(),
+    oldLang: string = null;
+
+  presence.on("UpdateData", async () => {
+    if (!document.querySelector("#root"))
+      return presence.setActivity({ largeImageKey: "logo" });
+
+    const [newLang, timestamps, cover] = await Promise.all([
+      presence.getSetting("lang").catch(() => "en"),
+      presence.getSetting("timestamps"),
+      presence.getSetting("cover")
+    ]);
+
+    oldLang ??= newLang;
+    if (oldLang !== newLang) {
+      oldLang = newLang;
+      strings = await getStrings();
+    }
+
+    const presenceData: PresenceData = {
+        largeImageKey: cover
+          ? document
+              .querySelector<HTMLImageElement>(
+                'div[class="player__track-cover"] img'
+              )
+              .src.replaceAll("230", "600")
```

Comment:
since when can you use image url's xD

Reply:
Good question. I tried it out and it worked xD

Reply:
lol cool

Reply:
Well congrats, you opened a whole new world to us, since this wasn't a thing for a real long time.